src/smclarify/util/__init__.py

```diff
@@ -69,7 +68,7 @@ def keys(_xs):
     dict_pdfs = seq(pdfs).map(dict).list()
 
     # result aligned lists
-    aligned_lists: List[List[np.array]] = [[] for x in range(num_pdfs)]
+    aligned_lists: List[List] = [[] for x in range(num_pdfs)]
```

Comment:
Why the reduced specificity?

Reply:
I debugged a unit test case (test_JS) and found that the aligned_lists is a list of list of float (instead of np.ndarray), so the typing annotation here was wrong or incomplete. From the context the type should be a scalar instead of array, but the typing information around the code is ambiguous, so I'm not sure if float is the only type. The purpose of the PR is to unblock build so I simply removed the incorrect annotation, we can revisit it later and add more specific typing annotations.

Reply:
Sounds good!